.github/auto_assign.yml

```
@@ -10,9 +10,7 @@ reviewers:
   - yomarion
   - kevindavee
   - alexandre-abrioux
-  - KolevDarko
   - leoslr
-  - olivier7delf
   - MantisClone
```

Consider adding recent hires.

```suggestion
  - MantisClone
  - skiv71
  - sstefdev
```